### PR TITLE
Makefile: remove unused targets

### PR DESCRIPTION
**Pull request by flavianmissi**
also refactor test_postgres target so it only starts postgres, without
running any tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,54 +74,25 @@ test_postgres:
...
-docker-build: build
-	ifneq (0,$(shell git status --porcelain | awk 'BEGIN {print $N}'))
-	echo 'dirty build not supported - run `FORCE=true make clean` to remove'
-	exit 1
-	endif
-	# get named head (ex: branch, tag, etc..)
-	NAME = $(shell git rev-parse --abbrev-ref HEAD)
-	# checkout commit so .git/HEAD points to full sha (used in Dockerfile)
-	git checkout $(SHA)
+	@echo $$(for i in {1..120}; do printf "-"; done)
```
[the remark this answers is not in the document]
Comment (flavianmissi) [reply Makefile:81]: huh, is it the loop range? I suppose I could use `seq` if that's the case

File: Makefile [hunk 2]
```diff
@@ -77,54 +74,25 @@ test_postgres:
...
+	@echo "postgres is ready to accept connections!"
+	@echo "now run pytest with the necessary env vars, i.e:"
+	@echo "'$(TEST_ENV) pytest <your-test-file.py>'"
+	@echo "attaching to postgres container, use Ctrl+C detach and stop the container"
+	@echo $$(for i in {1..120}; do printf "-"; done)
+	docker attach postgres-testrunner-postgres
```
Comment (dmage) [inline Makefile:87]: It used to run tests, and now it's an interactive command. Makefile isn't the best place for interactive shells

Comment (flavianmissi) [reply Makefile:87]: the reason why I removed the test run is that the whole suite takes over 40 min to run, so in my experience I usually run specific tests rather then the whole thing.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmage) [reply Makefile:81]:
```
$ bash -c 'echo {1..5}'
1 2 3 4 5
$ dash -c 'echo {1..5}'
{1..5}
```

I'd rather use
```
------------------------------------------------------------------------------------------------------------------------
```
without any for-loops. The most portable version :)

**Comment by flavianmissi** [comment]
we discussed this on slack and didn't really reach a conclusion on whether we want such a make target. closing this for now

**Pull request closed without merging**